drivers/richamster.js

```diff
@@ -0,0 +1,40 @@
+const Driver = require('../models/driver');
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat } = require('../lib/utils.js');
+
+/**
+ * @memberof Driver
+ * @augments Driver
+ */
+class Richamster extends Driver {
+  /**
+   * @augments Driver.fetchTickers
+   * @returns {Promise.Array<Ticker>} Returns a promise of an array with tickers.
+   */
+  async fetchTickers() {
+    const markets = await request('https://richamster.com/api/v1/ticker');
+    const marketsDetails = await request('https://richamster.com/public/v1/exchange/ticker/');
+
+    return Object.keys(markets).map((marketName) => {
+      const [base, quote] = marketName.split('_');
```

Comment:
If I fetch the tickers now, the symbols are separated by a '/'. They have probably changed this since you made this. Could you look into this?

Reply:
Oh my bad, took the wrong endpoint